packages/types-dev/index.d.ts

```diff
@@ -13,16 +13,38 @@ type AtLeastOne<T, Req = { [K in keyof T]-?: T[K] }, Opt = { [K in keyof T]+?: T
 
 declare global {
     namespace ioBroker {
-        enum StateQuality {
-            good = 0x00, // or undefined or null
```

Comment:
but this is "breaking"? or? should we leave both for now?

Reply:
Only on type level, it’s not a real enum which could have been used by someone in executed code.

Reply:
ok, then